dashboard/test/models/resource_test.rb

```diff
@@ -5,4 +5,11 @@ class ResourceTest < ActiveSupport::TestCase
     resource = create :resource, key: 'my key'
     assert_equal 'my key', resource.key
   end
+
+  test "resource can belong to lessons" do
+    resource = create :resource, key: 'my key'
```

Comment:
I would recommend creating more than one resource here, to make sure that the key sequence is working without violating the unique constraint.